changelogs/fragments/1645-proxmox-env-passthrough.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - proxmox - Add environment variable passthrough to Proxmox inventory module (https://github.com/ansible-collections/community.general/pull/1645).
```

Comment:
```suggestion
  - proxmox inventory plugin - add environment variable passthrough (https://github.com/ansible-collections/community.general/pull/1645).
```

Reply:
Addressed in https://github.com/ansible-collections/community.general/pull/1645/commits/1cc7e1c64c8a59ff691a658b5fbe237ed89a120c